external-api/src/database/mongo-client.ts

```diff
@@ -1,6 +1,8 @@
-const { MongoClient } = require('mongodb'); // eslint-disable-line @typescript-eslint/no-var-requires
+// eslint-disable-line @typescript-eslint/no-var-requires
```

Comment:
Eslint comment is not required or should be moved to other place.

Reply:
Have removed.